.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
+}
```

Comment:
Please add a newline here.

Reply:
Adding this file was a mistake. Removed it in the final commit.